mysticeti-core/src/commit_observer.rs

```diff
@@ -180,3 +180,67 @@ impl<H: ProcessedTransactionHandler<TransactionLocator> + Send + Sync> CommitObs
         self.commit_interpreter.committed = committed;
     }
 }
+
+pub struct SimpleCommitObserver<H = HashSet<TransactionLocator>> {
+    commit_interpreter: Linearizer,
+    transaction_votes: TransactionAggregator<QuorumThreshold, H>,
```

Comment:
We should remove this for now, since we are not using fast path on Mysticeti side